app/src/main/java/com/isc/hermes/MainActivity.java

```diff
@@ -56,14 +56,15 @@ public class MainActivity extends AppCompatActivity implements OnMapReadyCallbac
     private String mapStyle;
     private CurrentLocationController currentLocationController;
     private FilterCategoriesController filterCategoriesController;
-    private boolean visibilityMenu = false;
     private TextView searchView;
     private SharedSearcherPreferencesManager sharedSearcherPreferencesManager;
     private MarkerManager markerManager;
     private ActivityResultLauncher<Intent> launcher;
     private DrawerLayout drawerLayout;
     private NavigationView navigationView;
     private Toolbar toolbar;
+    private ImageButton buttonClear;
+    private String resetSearch = "Search...";
```

Comment:
Watching the usage of this variable, a more appropiate name should be "searchText"

Reply:
Done.